app/components/behandling/BehandlingCard.tsx

```diff
@@ -376,6 +426,13 @@ export default function BehandlingCard(props: Props) {
                   ) : (
                     <></>
                   )}
+                {props.behandling.planlagtStartet ? (
```

Comment:
Mange liker heller å gjøre sånt for å slippe den ekstra elsen: 


```suggestion
                {props.behandling.planlagtStartet && (
                   <Entry labelText={'Planlagt kjøring frem i tid'}>
                    {formatIsoTimestamp(props.behandling.planlagtStartet)}
                   </Entry>
                )
```